tools/readme-catalog/gen-audit-catalog.js

```diff
@@ -0,0 +1,62 @@
+import * as fs from 'fs';
```

Comment:
Copyright statement? (I realise we're somewhat inconsistent for scripts...)

Reply:
Yeah, will add. Good point.

Maybe there's a bot out there.

Reply:
Will be enforced with https://github.com/googleapis/google-cloudevents/pull/159